docs/octant-plugin-installation.md

```diff
@@ -87,7 +87,8 @@ You can follow the steps listed below to install octant and antrea-octant-plugin
     rpm -i octant_0.16.1_Linux-64bit.rpm
     ```
 
-2. Export your kubeconfig path (file location depends on your setup) to environment variable $KUBECONFIG.
+2. Export your kubeconfig path (file location depends on your setup) to environment variable $KUBECONFIG,
+   the default value is `~/.kube/config`
```

Comment:
`~/.kube/config` is not the default value of the environment variable but a fallback approach to discover kubeconfig. Should it say "otherwise `~/.kube/config` will be used."

Reply:
Sure